spec/site/tlc/clock_spec.rb

```diff
@@ -157,7 +157,7 @@
       Validator::Site.connected do |task,supervisor,site|
         prepare task, site
         with_clock_set CLOCK do
-          result = set_functional_position 'NormalControl'
+          result = set_functional_position 'NormalControl' 0
```

Comment:
i think you're missing a comma here between 'NormalControl' and 0, to separate arguments?
the same several place below